### PR TITLE
Add Jenkinsfile declarative pipeline step

### PR DESCRIPTION
**Pull request by 211217613**
add a snippet for linting your dockerfiles to a jenkinsfile

### CODE REVIEW

File: docs/INTEGRATION.md [hunk 1]
```diff
@@ -79,7 +79,27 @@ Add the following job to your project's `.drone.yml` pipeline:
...
+
+```groovy
+stage ("lint dockerfile") {
+    agent {
+        docker {
+            image 'hadolint/hadolint:latest-debian'
+        }
+    }
+    steps {
+        sh 'hadolint dockerfiles/* | tee -a hadolint_lint.txt'
```
Comment (CCFenner) [inline docs/INTEGRATION.md]: In the Jenkins environment the checkstyle format could be digested by the [warnings-ng plugin](https://plugins.jenkins.io/warnings-ng).

```suggestion
        sh 'hadolint dockerfiles/* -f checkstyle | tee -a hadolint_lint.xml'
```

File: docs/INTEGRATION.md [hunk 2]
```diff
@@ -79,7 +79,27 @@ Add the following job to your project's `.drone.yml` pipeline:
...
+    }
+    post {
+        always {
+            archiveArtifacts 'hadolint_lint.txt'
```
Comment (CCFenner) [inline docs/INTEGRATION.md]: along with the suggestion above publish the check style report with warnings-ng plugin
```suggestion
            archiveArtifacts 'hadolint_lint.xml'
            recordIssues blameDisabled: true, enabledForFailure: true, tools: [checkStyle(pattern: 'hadolint_lint.xml')]

```

File: docs/INTEGRATION.md (hunk 1 above)
Comment (lorenzo) [reply docs/INTEGRATION.md]: looks like a good idea

Comment (211217613) [reply docs/INTEGRATION.md]: I was also experimenting with `hadolint -f json dockerfiles/*` then publishing to a slack channel using 
```
slackSend(
    message: "Dockerfile Error",
    attachments: hadolint_lint.json
)
```

**Comment by lorenzo** [comment]
I will merge this as in itself it is a great addition. Thanks!

**Pull request merged**